config/settings/base.py

```diff
@@ -393,6 +393,9 @@
 FEATURE_COOKIE_BANNER_ENABLED = strtobool(
     os.getenv("FEATURE_COOKIE_BANNER_ENABLED", "True")
 )
+FEATURE_SHOW_STORIES_TOPIC_AND_TIME_PROMOS = strtobool(
+    os.getenv("FEATURE_SHOW_STORIES_TOPIC_AND_TIME_PROMOS", "True")
```

Comment:
Hey @jamesbiggs, 

Could you use the same convention of `THING_ENABLED` just to be consistent.